dev/tests/functional/tests/MFTF/DevDocs/Test/DevDocsTest.xml

```diff
@@ -48,8 +48,8 @@
         </actionGroup>
 
         <assertEqualsCanonicalizing stepKey="assertMergedArray">
-            <actualResult type="array">{{MessageData.numbers}}</actualResult>
-            <expectedResult type="array">["Something New", "0", "1", "2", "3"]</expectedResult>
+            <actualResult type="array">{{ExtendedMessageData.numbers}}</actualResult>
```

Comment:
I don't think we lose it.
In 1 assertion we cover both.
Because MessageData gets merged and then on top of it we merge `ExtendedMessageData.numbers`

Reply:
This is not a big deal. I am ok with either way.